app/views/messages/_message.html.erb

```diff
@@ -1,22 +1,27 @@
-<div id="<%= dom_id message %>">
-  <p class="my-5">
-    <strong class="block font-medium mb-1">Conversation:</strong>
-    <%= message.conversation_id %>
-  </p>
+<div id="<%= dom_id message %>" class="ml-9 md:ml-8 sm:ml-3 mb-5 group">
```

Comment:
I think I'd move your `mb-5` up to the flex container as `gap-5`. What do you think?

I think your `md:` and `sm:` classes should be reversed, so that the medium one overrides the small one. But more generally, splitting hairs over the left margin at different breakpoints has me wondering what this looks like through the different sizes.

Reply:
Good catch